sra_experiment/1_download_process_data.ipynb

```diff
@@ -0,0 +1,12781 @@
+      "[2020-07-08 15:53:46.779] [jointLog] [info] setting maxHashResizeThreads to 8\n",
+      "[2020-07-08 15:53:46.779] [jointLog] [info] Fragment incompatibility prior below threshold.  Incompatible fragments will be ignored.\n",
+      "[2020-07-08 15:53:46.779] [jointLog] [info] Usage of --validateMappings implies use of minScoreFraction. Since not explicitly specified, it is being set to 0.65\n",
+      "[2020-07-08 15:53:46.779] [jointLog] [info] Usage of --validateMappings implies a default consensus slack of 0.2. Setting consensusSlack to 0.35.\n",
+      "[2020-07-08 15:53:46.779] [jointLog] [info] parsing read library format\n",
+      "[2020-07-08 15:53:46.779] [jointLog] [info] There is 1 library.\n",
+      "-----------------------------------------\n",
+      "| Loading contig table | Time = 743.27 us\n",
+      "-----------------------------------------\n",
+      "size = 7452\n",
+      "-----------------------------------------\n",
+      "| Loading contig offsets | Time = 30.331 us\n",
+      "-----------------------------------------\n",
+      "-----------------------------------------\n",
+      "| Loading reference lengths | Time = 17.876 us\n",
+      "-----------------------------------------\n",
+      "-----------------------------------------\n",
+      "| Loading mphf table | Time = 539.96 us\n",
+      "-----------------------------------------\n",
+      "size = 5641451\n",
+      "Number of ones: 7451\n",
+      "Number of ones per inventory item: 512\n",
+      "Inventory entries filled: 15\n",
+      "-----------------------------------------\n",
+      "| Loading contig boundaries | Time = 7.4043 ms\n",
+      "-----------------------------------------\n",
+      "size = 5641451\n",
+      "-----------------------------------------\n",
+      "| Loading sequence | Time = 486.44 us\n",
+      "-----------------------------------------\n",
+      "size = 5417921\n",
+      "[2020-07-08 15:53:46.830] [jointLog] [info] Loading pufferfish index\n",
+      "[2020-07-08 15:53:46.830] [jointLog] [info] Loading dense pufferfish index.\n",
+      "-----------------------------------------\n",
+      "| Loading positions | Time = 5.3835 ms\n",
+      "-----------------------------------------\n",
+      "size = 5626187\n",
+      "-----------------------------------------\n",
+      "| Loading reference sequence | Time = 480.01 us\n",
+      "-----------------------------------------\n",
+      "-----------------------------------------\n",
+      "| Loading reference accumulative lengths | Time = 35.581 us\n",
+      "-----------------------------------------\n",
+      "[2020-07-08 15:53:46.845] [jointLog] [info] done\n",
+      "[2020-07-08 15:53:46.845] [jointLog] [info] Index contained 5,685 targets\n",
+      "[2020-07-08 15:53:46.846] [jointLog] [info] Number of decoys : 0\n",
+      "\n",
+      "\n",
+      "\n",
+      "\n",
+      "[2020-07-08 15:53:47.759] [jointLog] [info] Automatically detected mos
```

Comment:
As an outsider, I would benefit from a little two sentence walkthrough of these graphs. What do blue and red represent? What do these results say about the quality of your data?

Reply:
Good point. This was a quick validation to check that i processed the data correctly. So I have two different strains and I expected to see that they separate, which they do